Tests/Shared.Specs/ExceptionAssertionSpecs.cs

```diff
@@ -1080,13 +1080,15 @@ public void When_no_exception_should_be_thrown_after_wait_time_but_it_was_it_sho
             //-----------------------------------------------------------------------------------------------------------
             // Arrange
             //-----------------------------------------------------------------------------------------------------------
-            var watch = Stopwatch.StartNew();
             var waitTime = 100.Milliseconds();
```

Comment:
With using the abstracted timer, concrete time value should not be used anymore.
The expected behavior should be fully controlled by the test.

Reply:
You resolved the "conversation" without conversation.
:(
It seems you have no interest in my thoughts.

Reply:
Sorry. I must have missed that one.

Reply:
Wait, what part did I forget? That file is not using the `Stopwatch` anymore.

Reply:
The confusion my be related to the fact that I just selected one line for an comment which is intended to be more an overall comment:
You have introduced here the "new fake" for timing. I expect that specific time should NOT BE used anymore.
My understanding is that the fake completely remove any times from the test code.
Maybe, internally the specified time values will be not be used anymore for waiting or similar. But this is only understood by reading the implementation code of FakeClock.
Your attempt to "simulate time" by "incrementing now" looks interesting, but reading here "waittime" I expect the test code will "sleep" for the waittime. If it does not really wait "you" may be confused.
Hope this clarifies my thoughts...

Reply:
Yes, but I'm going to postpone that for a next PR. For now, my goal is to fix the stability issue with that specific set of tests that is now causing all PRs to be red.